recipe/meta.yaml

```diff
@@ -15,14 +15,15 @@ source:
 
 build:
   number: {{ build }}
-  string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+  skip: True  # [win or py2k]
+  string: "{{ processor }}_h{{ PKG_HASH }}_{{ build }}"
+
+requirements:
+  host:
+    - python
 
 outputs:
   - name: prismatic
-    build:
-      number: {{ build }}
-      skip: True  # [win]
-      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
     requirements:
```

Comment:
The build information here is needed, because this script only creates the subpackages. It is a different setup to most conda-forge meta.yaml files - but it is the setup I found that worked.